### DISCUSSION TITLE
Dual layout (dual grid)

### DISCUSSION THREAD

**Discussion by jannick-ux** [SchwarzIT/onyx#138] [Ideas]
<img width="2025" alt="Bildschirm­foto 2024-01-17 um 09 10 18" src="https://github.com/SchwarzIT/onyx/assets/152376364/0a68e9ae-94c0-49ac-8410-026266e5426c">
How about having the possibility to display two whole different sections next to each other on a larger screen (separated by a vertical separator). Different content and different scroll container should be possible as well as the implementation of a "additional grid" for the second section.

**Comment by mj-hof** [comment]
This feature is provide by #2262
